libs/src/services/identity/index.js

```diff
@@ -26,6 +30,9 @@ class IdentityService {
   }
 
   async setUserFn (obj) {
+    const token = await this.captcha.generate('identity/user')
```

Comment:
do we want to also add captcha score calculation here too? @raymondjacobson

Reply:
nah, i think one endpoint is fine for starters

Reply:
discussed in closed PR -- will move auth recaptcha work into the user route